src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/ILspLogger.cs

```diff
@@ -8,12 +8,7 @@
 using System;
 
 namespace Microsoft.CommonLanguageServerProtocol.Framework;
-
-#if BINARY_COMPAT // TODO - Remove with https://github.com/dotnet/roslyn/issues/72251
-public interface ILspLogger
-#else
 internal interface ILspLogger
```

Comment:
>internal [](http://example.com/codeflow?start=0&length=8)

nit: another here, will stop marking these

Reply:
I have one or two more followups in this area, so I'll defer these until then.